install/jakartaee/bin/sig-test-pkg-list.txt

```diff
@@ -132,17 +132,17 @@ javax.servlet.jsp.jstl.tlv
 javax.servlet.jsp.tagext
 javax.transaction
 javax.transaction.xa
-javax.validation
-javax.validation.Configuration<{javax.validation
-javax.validation.Configuration<{javax.validation.bootstrap
-javax.validation.Configuration<{javax.validation.spi
-javax.validation.bootstrap
-javax.validation.constraints
-javax.validation.constraintvalidation
-javax.validation.executable
-javax.validation.groups
-javax.validation.metadata
-javax.validation.spi
+jakarta.validation
+jakarta.validation.Configuration<{javax.validation
```

Comment:
jakarta.validation.Configuration<{javax.validation shouldn't this be jakarta.validation.Configuration<{jakarta.validation ?

Reply:
thanks, @gurunrao for reviewing this PR. I have updated this PR with require changes.